frontend/src/pages/ActivityReport/Pages/components/GoalPicker.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Goal from './Goal';
+import MultiSelect from '../../../../components/MultiSelect';
+
+import Option from './GoalOption';
+import Input from './GoalInput';
+
+const components = {
+  Input,
+  Option,
+};
+
+const GoalPicker = ({
+  control, availableGoals, selectedGoals, setValue,
+}) => {
+  const onRemove = (id) => {
+    const newGoals = selectedGoals.filter((selectedGoal) => selectedGoal.id !== id);
+    setValue('goals', newGoals);
```

Comment:
How does `setValue` work here? From the [docs](https://react-hook-form.com/api#setValue)  it says it needs to reference a `registered` field but I can't seem to see where that field exists here?

Reply:
The "goals" field is "registered" in the multiselect component because the multiselect component uses [controller](https://react-hook-form.com/api#Controller) from `react-hook-from`

Reply:
I love these comments linking to the relevant docs.